node/make.js

```diff
@@ -38,6 +38,7 @@ target.test = function() {
     buildutils.getExternals();
     run('tsc -p ./test');
     cp('-Rf', rp('test/scripts'), testPath);
+    process.env['TASKLIB_INPROC_UNITS'] = '1'; // export task-lib internals for internal unit testing
```

Comment:
i changed it up so `_internal` is only exported if this env var is set, which means it needs to be set before the first `require('task')`. before launching the mocha process is the easiest way to do that.